File: src/main/java/me/zimy/questionnaire/configuration/MVCConfig.java
package me.zimy.questionnaire.configuration;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

import static org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration.WebMvcAutoConfigurationAdapter;

/**
 * Ignore this, do not touch it at all. That component allows to serve static content.
 *
 * @author Dmitriy &lt;Zimy&gt; Yakovlev
 * @since 12/1/14.
 */

@EnableWebMvc
@Configuration
public class MVCConfig extends WebMvcAutoConfigurationAdapter {
}

File: src/main/kotlin/me/zimy/questionnaire/configuration/MVCConfig.kt
package me.zimy.questionnaire.configuration

import org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration.WebMvcAutoConfigurationAdapter
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.EnableWebMvc

/**
 * Ignore this, do not touch it at all. That component allows to serve static content.

 * @author Dmitriy &lt;Zimy&gt; Yakovlev
 * *
 * @since 12/1/14.
 */

@EnableWebMvc
@Configuration
open class MVCConfig : WebMvcAutoConfigurationAdapter()
